aten/src/ATen/native/mps/MPSGraphVenturaOps.h

```diff
@@ -89,4 +89,26 @@ typedef NS_ENUM(NSUInteger, MPSGraphResizeNearestRoundingMode)
                                     scaleOffsetTensor:(MPSGraphTensor * _Nonnull) scaleOffset
                                                layout:(MPSGraphTensorNamedDataLayout) layout
                                                  name:(NSString * _Nullable) name;
+
+- (MPSGraphTensor * _Nonnull) sampleGridWithSourceTensor:(MPSGraphTensor * _Nonnull) source
+                               coordinateTensor:(MPSGraphTensor * _Nonnull) coordinates
```

Comment:
Can you fix the indents? I got this comment with a similar issue:
https://github.com/pytorch/pytorch/pull/91669#discussion_r1061045475

Reply:
Thanks - I'll update this.